docs/UserGuide.adoc

```diff
@@ -542,52 +542,51 @@ then annotations are shown as well.
 
 === Adding reminder: `reminder`
 
-Adds a reminder to visit a bookmark. You can set the time for the reminder. The app
-will inform you at that time. You can also choose whether to open the bookmark.
+Adds a reminder to visit a bookmark. You can set the time for the reminder.
+You can also choose whether to open the bookmark.
 The default setting is open.
 
 Format: `reminder INDEX t/TIME_DATE [n/NOTE=Open]`
 
 * INDEX is the index of the bookmark that you want to add with a reminder.
  * TIME_DATE follows HHMM dd/mm/yyyy format.
-* NOTE is the setting of whether you want to open the bookmark.
+* NOTE is the description of the reminder, the default value is Open.
 
 Examples:
 
 * `reminder 1 t/2359 02/07/2019` +
-Adds a reminder for bookmark 1, and set the time at 02/07/2019. The
-reminder will remind you at 23:59 on 2nd July, 2019.
-It will also open the bookmark for you.
+Adds a reminder for bookmark 1, and sets the time of the reminder as 23:59 of 02/07/2019.
+It reminds you to open the bookmark.
+
+* `reminder 1 t/2359 02/07/2019 n/Read announcements` +
+Adds a reminder for bookmark 1, and sets the time of the reminder as 23:59 of 02/07/2019.
+The reminder reminds you to read announcements.
+
 
-* `reminder 1 t/2359 02/07/2019 n/Close` +
-Adds a reminder for bookmark 1, and set the time at 23:59 of 02/07/2019. The reminder will
-remind you at the time and would not open the bookmark for you.
 
 === Editing reminder: `reminder edit`
 Edit a reminder at the reminder's index. You can edit either time or whether to open it.
 
 Format: `reminder edit INDEX [t/TIME_DATE] [n/NOTE]`
-* INDEX is the index of the reminder that you want to edit.
+* INDEX is the index of the bookmark that has the reminder that you want to edit.
 * TIME_DATE is the time you want to edit. The time follows HHMM dd/mm/yyyy format.
-* NOTE is the setting that you want to edit. It is whether you want to open the bookmark.
-
+* NOTE is the description of the reminder that you want to edit.
 * `reminder edit 1 t/2359 02/07/2019` +
-Edit reminder in index 1. Reset the time of the reminder to 23:59 of 02/07/2019.
-
-* `reminder 1 t/2359 02/07/2019 n/Close` +
-Edit reminder in index 1. Reset the time of the reminder to 23:59 of 02/07/2019.
-Reset to not to open the bookmark.
+Edit reminder of bookmark in index 1. Reset the time of the reminder to 23:59 of 02/07/2019.
```

Comment:
```suggestion
Edits the reminder of bookmark 1. Reset the time of the reminder to 23:59 of 02/07/2019.
```

Reply:
@TSAI-HSIAO-HAN 

FYI, you can commit all of the suggested changes directly from GitHub:
1. Go to the tab `Files Changed`
2. Click `Add suggestion to batch` for each suggested change. 
3. Click `Commit` (or whatever the button is named) once you're done adding all the changes.